.github/workflows/release-winget.yml

```diff
@@ -0,0 +1,13 @@
+name: Publish release to WinGet
+on:
+  release:
+    types: [released]
+jobs:
+  publish:
+    runs-on: windows-latest
+    steps:
+      - uses: vedantmgoyal9/winget-releaser@latest
+        with:
+          identifier: LRCLIB.LRCGET
+          installers-regex: 'x64_en-US\.msi '
```

Comment:
Just to be sure, is the space at the end after `.msi` intentional?

Reply:
No sorry about that